src/main/java/Commands/Implementations/CommandImpl.java

```diff
@@ -0,0 +1,6 @@
+package Commands.Implementations;
+
+import java.util.HashMap;
+
+public class CommandImpl extends HashMap<String, String> {
+}
```

Comment:
For now, (if this is a good enough interface) we do not even need this class. Just a plain old `HashMap` would work. The name is already tied to the interface.